tskv/tests/test_kvcore_interface.rs

```diff
@@ -129,8 +183,12 @@ mod tests {
                 test_kvcore_snapshot_create_apply_delete();
             })
             .await;
+
+            res
```

Comment:
exec **make clippy**
then exec **make fmt**

Reply:
done it